desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -419,6 +419,11 @@ def close_session(self, session):
 
   @query_error_handler
   def autocomplete(self, snippet, database=None, table=None, column=None, nested=None, operation=None):
+    if snippet['type'] == 'phoenix':
```

Comment:
This one could you make a little python script to demo the problem and ask the phoenix team if this is something that you be fixed in the pyphoenix connector?

```from sqlalchemy import inspect
from sqlalchemy.engine import create_engine
engine = create_engine('phoenix://localhost:6875')

..get_table_names(database)
...get_columns(table, database)
```

https://docs.gethue.com/administrator/configuration/connectors/#apache-phoenix

Reply:
What type of python script? I am not getting it.